spec.html

```diff
@@ -32072,92 +32072,283 @@ <h1>Time Values and Time Range</h1>
+          <dd>It returns the number of milliseconds since the start of the day in which _t_ falls.</dd>
+        </dl>
+        <emu-alg>
+          1. Return 𝔽(ℝ(_t_) modulo ℝ(msPerDay)).
+        </emu-alg>
+      </emu-clause>
+
+      <emu-clause id="sec-daysinyear" type="abstract operation" oldids="eqn-DaysInYear,sec-year-number">
+        <h1>
+          DaysInYear (
+            _y_: an integral Number,
+          ): *365*<sub>𝔽</sub> or *366*<sub>𝔽</sub>
+        </h1>
+        <dl class="header">
+          <dt>description</dt>
+          <dd>It returns the number of days in year _y_. Leap years have 366 days; all other years have 365.</dd>
+        </dl>
+        <emu-alg>
+          1. Let _ry_ be ℝ(_y_).
+          1. If (_ry_ modulo 400) = 0, return *366*<sub>𝔽</sub>.
+          1. If (_ry_ modulo 100) = 0, return *365*<sub>𝔽</sub>.
+          1. If (_ry_ modulo 4) = 0, return *366*<sub>𝔽</sub>.
+          1. Return *365*<sub>𝔽</sub>.
+        </emu-alg>
+      </emu-clause>
+
+      <emu-clause id="sec-dayfromyear" type="abstract operation" oldids="eqn-DaysFromYear">
+        <h1>
+          DayFromYear (
+            _y_: an integral Number,
+          ): an integral Number
+        </h1>
+        <dl class="header">
+          <dt>description</dt>
+          <dd>It returns the day number of the first day of year _y_.</dd>
+        </dl>
+        <emu-alg>
+          1. Let _ry_ be ℝ(_y_).
+          1. NOTE: In the following steps, each `_numYearsN_` is the number of years divisible by N that occur between the epoch and the start of year _y_. (The number is negative if _y_ is before the epoch.)
+          1. Let _numYears1_ be (_ry_ - 1970).
+          1. Let _numYears4_ be floor((_ry_ - 1969) / 4).
+          1. Let _numYears100_ be floor((_ry_ - 1901) / 100).
+          1. Let _numYears400_ be floor((_ry_ - 1601) / 400).
+          1. Return 𝔽(365 × _numYears1_ + _numYears4_ - _numYears100_ + _numYears400_).
+        </emu-alg>
+      </emu-clause>
+
+      <emu-clause id="sec-timefromyear" type="abstract operation" oldids="eqn-TimeFromYear">
+        <h1>
+          TimeFromYear (
+            _y_: an integral Number,
+          ): a time value
+        </h1>
+        <dl class="header">
+          <dt>description</dt>
+          <dd>It returns the time value of the start of year _y_.</dd>
+        </dl>
+        <emu-alg>
+          1. Return msPerDay × DayFromYear(_y_).
+        </emu-alg>
+      </emu-clause>
+
+      <emu-clause id="sec-yearfromtime" type="abstract operation" oldids="eqn-YearFromTime">
+        <h1>
+          YearFromTime (
+            _t_: a finite time value,
+          ): an integral Number
+        </h1>
+        <dl class="header">
+          <dt>description</dt>
+          <dd>It returns the year in which _t_ falls.</dd>
+        </dl>
+        <emu-alg>
+          1. [declared="y"] Return the largest integral Number _y_ (closest to +∞) such that TimeFromYear(_y_) ≤ _t_.
```

Comment:
```suggestion
          1. [declared="y"] Return the largest integral Number _y_ (closest to *+∞*<sub>𝔽</sub>) such that TimeFromYear(_y_) ≤ _t_.
```

As mentioned in editor call today. We can hold it for a separate PR too, if you want.